Migrate OpenVPN integration guides from the emnify Knowledge Base (#238)
emnify/product-docs: docs/how-tos/openvpn/integration-linux.md

---
description: Integration Guide for OpenVPN on Linux
last_update: 

>   date: 01-11-2024

Reviewer: These dates should match the dates on the KB article.

```suggestion
  date: 11-23-2022
```

slug: /how-tos/openvpn-integration-linux
---

# OpenVPN Integration Guide for Linux

emnify customers can create their own Virtual Private Network for their mobile IoT/M2M devices fitted with emnify SIMs.
Data traffic will be exchanged between the devices and the application server through an OpenVPN tunnel, enabling direct communication with the IPs of the mobile devices (no NAT applied).

The tunnel is established between the emnify Core Network and the customers VPN gateway or server.

See this video [guide for how to secure your devices with Open VPN](https://www.youtube.com/watch?v=yt44fJpfkQ4).

> ## Add mandatory layer of security and privacy

Reviewer: ```suggestion
## Configure OpenVPN in the emnify Portal
```

Any traffic exchanged with the mobile devices is encrypted before transmitted over the public internet, therefore adding an additional layer of security and privacy.
For that **no** VPN software needs to be installed on the device or any configuration changes to be done, the default emnify APN does also support VPN flows.

> First, download the VPN config file from the emnify User Interface

Reviewer: FYI for all migrations: Any references to the "EUI" or "emnify User Interface" need to be changed to Portal (for functionality that exists in the Portal, any actual EUI topics shouldn't be migrated into the docs yet).

```suggestion
First, download the VPN configuration file from the emnify Portal.
```
